examples/ex6.cpp

```diff
@@ -20,7 +20,8 @@
 //               ex6 -pa -d occa-cuda
 //               ex6 -pa -d raja-omp
 //               ex6 -pa -d ceed-cpu
-//               ex6 -pa -d ceed-cuda
+//               * ex6 -pa -d ceed-cuda
+//               ex6 -pa -d ceed-cuda:/gpu/cuda/ref
```

Comment:
Is this the best backend to use, what about `/gpu/cuda/shared`?

Reply:
Good point, thank you.